procedures/unit-testing-basics.ipf

```diff
@@ -2245,6 +2289,43 @@ static Function AfterTestCase(name)
 
 	DFREF dfr = GetPackageFolder()
 	NVAR/SDFR=dfr assert_count
+	NVAR/SDFR=dfr waveTrackingMode
+
+#if IgorVersion() >= 9.0
+	if(NVAR_Exists(waveTrackingMode))
+		if((waveTrackingMode & UTF_WAVE_TRACKING_LOCAL) == UTF_WAVE_TRACKING_LOCAL)
+			WaveTracking/LOCL count
+			if(V_Flag == WAVE_TRACKING_COUNT_MODE)
+				if(V_numWaves)
+					sprintf msg, "Local wave leak detected (leaked waves: %d) in \"%s\"", V_numWaves, name
+					TestCaseFail(msg)
+				endif
+				WaveTracking/LOCL stop
+			elseif(V_Flag != WAVE_TRACKING_INACTIVE_MODE)
+				// do nothing for WAVE_TRACKING_INACTIVE_MODE.
```

Comment:
This creates some unintuitive situation. If you add on purpose to TestProduceWaveLeaks
`WaveTracking/FREE tracker` then the UTF fails for this TC, but also the next TC TestWaveLeaksIgnored fails.
Though, how can this happen when it is marked with `// UTF_NO_WAVE_TRACKING` ?

So if waveTrackingMode is enabled you have to either stop any user induced WaveTracking after the TC or
you stop it in BeforeTestCase even if the tag is set (when waveTrackingMode is enabled).